src/sass/SimpleSwitch.scss

```diff
@@ -73,6 +74,12 @@ $simple-switch_tray-color: #ccc !default;
         }
     }
 
+    &.disabled {
+        background-color: $simple-switch_disable-color;
+        border-color: $simple-switch_disable-color;
+        pointer-events: none;
```

Comment:
NIT: `pointer-events: none;` allows clicks to fall through to the parent element (or generally, the element "behind" this element). This *could* introduce some weird functionality. It's probably better to change the mouse cursor and ignore click events than use this CSS property, from an accessibility and interop standpoint.

Reply:
Hmm... You mean ignore the click events via JS? I could add code to do that

Reply:
Yes, that was my thought.